### PR TITLE
made updates for 0.6.2

### PR DESCRIPTION
**Pull request by ellesmith88**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ellesmith88** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ requirements:
   host:
     - python
     - pip
-    - pytest-runner
+    - pytest-runnerß
```
Comment (cehbrecht) [inline recipe/meta.yaml:21]: typo?

**Review by cehbrecht — COMMENTED** [review]
usually a PR gets triggered when there is a new version on pypi ... but it takes some time :)

Have the requirements changed?

File: recipe/meta.yaml (hunk 1 above)
Comment (ellesmith88) [reply recipe/meta.yaml:21]: > usually a PR gets triggered when there is a new version on pypi ... but it takes some time :)
> 
> Have the requirements changed?

I know, I thought I would do it manually as it was taking a while :) 

This happened when I rerendered - I didn't change it so I assume it's fine?

**Comment by ellesmith88** [comment]
> usually a PR gets triggered when there is a new version on pypi ... but it takes some time :)
> 
> Have the requirements changed?

The requirements haven't changed

File: recipe/meta.yaml (hunk 1 above)
Comment (ellesmith88) [reply recipe/meta.yaml:21]: Oh but it doesn't like it, I'll take it out

**Comment by ellesmith88** [comment]
Duplicated

**Pull request closed without merging**